app/scripts/create-admin.ts

```diff
@@ -0,0 +1,43 @@
+import { db } from "@/models";
+import env from "@next/env";
+import { randomUUID } from "node:crypto";
+import { logger } from "@/services/logger";
+import bcrypt from "bcrypt";
+import { Roles } from "@/lib/auth";
+
+async function createAdmin() {
+  const projectDir = process.cwd();
+  env.loadEnvConfig(projectDir);
+
+  if (!db.initialized) {
+    await db.initialize();
+  }
+
+  if (!process.env.DEFAULT_ADMIN_EMAIL || !process.env.DEFAULT_ADMIN_PASSWORD) {
+    logger.error(
+      "create-admin.ts: Missing default admin credentials DEFAULT_ADMIN_EMAIL and DEFAULT_ADMIN_PASSWORD in env!",
+    );
+    return;
+  }
+
+  const passwordHash = await bcrypt.hash(
+    process.env.DEFAULT_ADMIN_PASSWORD,
+    12,
+  );
+  const user = await db.models.User.create({
```

Comment:
I don't think this is idempotent. If this runs every time we deploy, it's going to cause an error, isn't it?

Reply:
It would fail the second time it's run, yes. I was going to have it as a one-off task instead of running it on each deploy, but I can make it check if the account already exists and exit gracefully in case that's important.